plugin/parser/engine/pg/test-data/test_drop_schema_data.yaml

```diff
@@ -0,0 +1,5 @@
+- stmt: drop schema myschema
+  want: DROP SCHEMA "myschema" RESTRICT
+
+- stmt: drop schema if exists s1, s2 cascade
+  want: DROP SCHEMA IF EXISTS "s1", "s2" CASCADE
```

Comment:
How about writing `;` and `\n` at the end of the SQL statement?

Reply:
I think this is the responsible of caller.

Reply:
I agree with the `\n` may be the responsibility of the caller. But I think need `;` is the most cases.

Reply:
I found that the PostgreSQL server-side parser doesn’t support `DELIMITER` statement. So writing `;` SGTM.
If we want to write the `;` at the end of the deparse result SQL statement. We can do it in the following PR.